contracts/proxy/transparent/TransparentUpgradeableProxy.sol

```diff
@@ -50,32 +50,6 @@ contract TransparentUpgradeableProxy is ERC1967Proxy {
         }
     }
 
-    /**
-     * @dev Returns the current admin.
-     *
-     * NOTE: Only the admin can call this function. See {ProxyAdmin-getProxyAdmin}.
-     *
-     * TIP: To get this value clients can read directly from the storage slot shown below (specified by EIP1967) using the
-     * https://eth.wiki/json-rpc/API#eth_getstorageat[`eth_getStorageAt`] RPC call.
-     * `0xb53127684a568b3173ae13b9f8a6016e243e63b6e8ee1178d6a717850b5d6103`
```

Comment:
We should make sure we are keeping this documentation somewhere. We may have to move it to the top-level contract docs.

Reply:
I moved both to the functions they were calling, `_getAdmin` and `_implementation`